components/resources/library/src/jsMain/kotlin/org/jetbrains/compose/resources/Resource.js.kt

```diff
@@ -17,28 +19,76 @@ import org.w3c.xhr.XMLHttpRequestResponseType
 import kotlin.coroutines.resume
 import kotlin.coroutines.resumeWithException
 import kotlin.coroutines.suspendCoroutine
+import kotlin.js.Promise
 
+/**
+ * WebResourcesConfiguration is used in [configureWebResources].
+ */
+@Suppress("unused")
+@ExperimentalResourceApi
+class WebResourcesConfiguration internal constructor() {
```

Comment:
Maybe 'object' here is better?

Reply:
Yes, it makes sense to use object here. Fixed